src/xenia/base/clock_win.cc

```diff
@@ -37,7 +35,7 @@ uint64_t Clock::QueryHostSystemTime() {
 }
 
 uint64_t Clock::QueryHostUptimeMillis() {
-  return QueryHostTickCount() / (host_tick_frequency() / 1000);
+  return host_tick_count_platform() / (host_tick_frequency_platform() / 1000);
```

Comment:
```suggestion
  return (host_tick_count_platform() * 1000) / host_tick_frequency_platform();
```

As we discussed previously on Discord, I would consider arithmetic on frequency risky, as you cannot really make assumptions about its units. I'll feel much safer when it's done via multiplication instead.

Reply:
Same for any other instances of dividing frequency, whether they are touched by your PR or not.